### PR TITLE
Added SSL to Oracle Assess Migration

### PR DESCRIPTION
**Pull request by shubham-yb**

### CODE REVIEW

File: migtests/scripts/functions.sh [hunk 1]
```diff
@@ -684,19 +684,24 @@ EOF
...
 	"
 	if [ "${SOURCE_DB_SCHEMA}" != "" ]
 	then
 		args="${args} --source-db-schema ${SOURCE_DB_SCHEMA}"
 	fi
+	if [ "${SOURCE_DB_ORACLE_TNS_ALIAS}" != "" ]
+	then
+		args="${args} --oracle-tns-alias ${SOURCE_DB_ORACLE_TNS_ALIAS}"
+	else
+		args="${args} --source-db-host ${SOURCE_DB_HOST} --source-db-port ${SOURCE_DB_PORT}"
```
Comment (sanyamsinghal) [inline migtests/scripts/functions.sh:703]: `--source-db-port` is already being considered above

File: migtests/scripts/functions.sh [hunk 2]
```diff
@@ -684,19 +684,24 @@ EOF
...
 		--send-diagnostics=false --yes
 		--start-clean t
-		--iops-capture-interval 0
+		--iops-capture-interval 1
 	"
 	if [ "${SOURCE_DB_SCHEMA}" != "" ]
 	then
 		args="${args} --source-db-schema ${SOURCE_DB_SCHEMA}"
 	fi
+	if [ "${SOURCE_DB_ORACLE_TNS_ALIAS}" != "" ]
```
Comment (sanyamsinghal) [inline migtests/scripts/functions.sh:699]: Question: If TNS_ALIAS is provided, does SCHEMA will still be required?

**Review by sanyamsinghal — CHANGES_REQUESTED** [review]

File: migtests/scripts/functions.sh (hunk 2 above)
Comment (shubham-yb) [reply migtests/scripts/functions.sh:699]: Ideally schema shouldn't be required. But there's other dependencies in our test structure for it. So it's kept that way for now. 
We can restructure it later if need be.

**Comment by shubham-yb** [comment]
Added in another PR
https://github.com/yugabyte/yb-voyager/pull/1601

**Pull request closed without merging**